react/features/video-menu/components/web/VideoMenuButton.js

```diff
@@ -23,7 +23,7 @@ type Props = {
     /**
      * The icon that will display within the component.
      */
-    icon: Object,
+    icon?: Object,
```

Comment:
Won't all items have an icon?

Reply:
We don't have an icon for the flip menu....